src/client/objects/WaivioObject.js

```diff
@@ -7,15 +7,13 @@ import WeightTag from '../components/WeightTag';
 import FollowButton from '../widgets/FollowButton';
 import ObjectAvatar from '../components/ObjectAvatar';
 import { addressFields, objectFields, websiteFields } from '../../common/constants/listOfFields';
-import { AppSharedContext } from '../Wrapper';
-import { getClientWObj } from '../adapters';
 
 import './WaivioObject.less';
 
 const WaivioObject = ({ wobj, unfollow, follow }) => {
-  const { usedLocale } = useContext(AppSharedContext);
-  const wObject = getClientWObj(wobj, usedLocale);
-  const { address, default_name: defaultName, name, website } = wObject;
+  const { default_name: defaultName, name } = wobj;
+  const address = wobj.address && JSON.parse(wobj.address);
+  const website = wobj.website && JSON.parse(wobj.website);
```

Comment:
Add error handling - JSON.parse() is not safe operation
You can create helper function parseWobjectField(wobject, fieldName)